homeassistant/components/improv_ble/config_flow.py

```diff
@@ -405,7 +405,7 @@ async def _try_call(func: Coroutine[Any, Any, _T]) -> _T:
             raise AbortFlow("characteristic_missing") from err
         except improv_ble_errors.CommandFailed:
             raise
-        except Exception as err:  # pylint: disable=broad-except
+        except Exception as err:
```

Comment:
Question: Why can these `# pylint: disable=broad-except` be removed?

Reply:
The exception is re-raised a few lines below, so the broad `except` here doesn't really impact anything. It's just used to add a logging statement.